apps/reference/docs/guides/platform/logs.mdx

```diff
@@ -4,78 +4,198 @@ title: Logging
 
-For example, you may enter the following into the SQL editor to query for each user's IP address:
+Raw top-level timestamp values are rendered as unix microsecond. To render the timestamps in a human-readable format, use the `DATETIME()` function to convert the unix timestamp display into an ISO-8601 timestamp.
 
 ```sql
-SELECT timestamp, h.x_real_ip
-FROM edge_logs
-  LEFT JOIN UNNEST(metadata) as m ON TRUE
-  LEFT JOIN UNNEST(m.request) AS r ON TRUE
-  LEFT JOIN UNNEST(r.headers) AS h ON TRUE
-WHERE h.x_real_ip IS NOT NULL
-```
+-- timestamp column without datetime()
+select timestamp from ....
+--  1664270180000
 
-![SELECT query example](/img/guides/platform/logs/select-query.png)
+-- timestamp column with datetime()
+select datetime(timestamp) from ....
+-- 2022-09-27T09:17:10.439Z
+```
 
-The list of supported product sources can be found under the **Sources** dropdown.
+:::info
+The rendering behaviour as unix microsecond will be removed in the future.
+:::
 
 ### Unnesting Arrays
 
-To query the metadata in the above example, you can unnest the field and "join" the unnested data. Clicking on the log row shows that log metadata is stored as an array of objects.
+Each log event stores metadata an array of objects with multiple levels, and can be seen by selecting single log events in the Logs Explorer. To query arrays, use `unnest()` on each array field and add it to the query as a join. This allows you to reference the nested objects with an alias and select their individual fields.
 
-In order to query any value that is an array, we would need to `UNNEST()` that field and add it to the query as a join, thereby allowing us to reference the nested fields within the array.
+For example, to query the edge logs without any joins:
 
-:::caution
+```sql
+select timestamp, metadata from edge_logs t
+```
 
-Large projects may run into a `Resources Exceeded` memory limit error when selecting large objects with many nested keys. To avoid this error, select individual keys separately or reduce the queried date range.
-:::
+The resulting `metadata` key is rendered as an array of objects when rendered in the Logs Explorer. This can be better visualized in this diagram, where each square in a column represents a nested object:
 
-### Functions
+![Without Unnesting](/img/guides/platform/logs/unnesting-none.png)
 
-You may have also noticed from the above examples that we are able to use certain SQL functions within our queries. The querying engine currently supported is BigQuery, hence you may use [any valid BigQuery function](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators) within your query.
+We can perform a `cross join unnest()` to work with keys nested below the `metadata` key.
 
-#### Timestamp Behavior
+To query for a deeply nested value, we will have to add a join for each array level:
```

Comment:
Would this still be correct?
```suggestion
To query for a nested value, add a join for each level of the array:
```

Reply:
will keep the array wording, should be "each array level", as there are many nested arrays (i.e. arrrays within arrays within arrays)